test/ui-pytest/constants/wallet.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class DerivationPath(Enum):
+    CUSTOM = 'Custom'
+    ETHEREUM = 'Ethereum'
+    ETHEREUM_ROPSTEN = 'Ethereum Testnet (Ropsten)'
+    ETHEREUM_LEDGER = 'Ethereum (Ledger)'
+    ETHEREUM_LEDGER_LIVE = 'Ethereum (Ledger Live/KeepKey)'
```

Comment:
Please verify that you have one empty line at the end of all added files

Reply:
![image](https://github.com/status-im/status-desktop/assets/128374224/318f5d26-0b05-45f1-ae9a-11034e66c8f7)
As I understand GitHud shows this icon if you don't have it

Reply:
Added